### PR TITLE
Update release-notes.mdx

### PR DESCRIPTION
**Pull request by marikaner**
Update release notes for VDM releases

### CODE REVIEW

File: docs/js/release-notes.mdx [hunk 1]
```diff
@@ -45,6 +45,24 @@ These libraries are not Open Source, but free and distributed under the SAP Deve
...
 :::
 
+## Version 1.24.0 - January 18, 2021
+
+### Compatibility Notes
+
+- A few operations were removed from the some SAP S/4HANA Cloud 2102 services. These operations were never supported by the SAP S/4HANA system (like CRUD support for some entities), so no existing functionality is affected. The following services are affected:
+  - _Nota Fiscal – Create, Update_ - `API_LOGBR_NOTAFISCAL_SRV`
+  - _Purchase Contracts_ - `API_PURCHASECONTRACT_PROCESS_SRV`
+  - _Outbound Delivery (A2X)_ - `API_OUTBOUND_DELIVERY_SRV_0002`
```
Comment (jjtang1985) [inline docs/js/release-notes.mdx:55]: The package `@sap/cloud-sdk-vdm-purchasing-inforecord-service` was renamed to `@sap/cloud-sdk-vdm-purchasing-info-record-service`.
Perhaps we can also mention it.

File: docs/js/release-notes.mdx [hunk 2]
```diff
@@ -45,6 +45,24 @@ These libraries are not Open Source, but free and distributed under the SAP Deve
 
 :::
 
+## Version 1.24.0 - January 18, 2021
+
+### Compatibility Notes
+
```
Comment (jjtang1985) [inline docs/js/release-notes.mdx:51]: Have we received the doc link from Tobias, which should be added here.

**Review by jjtang1985 — CHANGES_REQUESTED** [review]

File: docs/js/release-notes.mdx [hunk 3]
```diff
@@ -45,6 +45,24 @@ These libraries are not Open Source, but free and distributed under the SAP Deve
...
+### New Functionality
+
+- **We released a regular update for pre-generated client libraries (also known as VDM) for the [latest RTC release 2102 of SAP S/4HANA Cloud](https://news.sap.com/2021/02/sap-s4hana-cloud-2102-empowering-businesses-seize-opportunities/).**
+  - The update covers all the changes to existing OData services and introduces new ones.
+- **We released a regular update for pre-generated client libraries (also known as VDM) for the latest RTC release 2020 FSP1 of SAP S/4HANA On-Premise.**
+  - The update covers all the changes to existing OData services and introduces new ones.
+
+### Improvements
+
```
Comment (marikaner) [inline docs/js/release-notes.mdx:65]:
```suggestion
```

File: docs/js/release-notes.mdx [hunk 4]
```diff
@@ -45,6 +45,24 @@ These libraries are not Open Source, but free and distributed under the SAP Deve
 
 :::
 
+## Version 1.24.0 - January 18, 2021
+
```
Comment (marikaner) [inline docs/js/release-notes.mdx:49]:
```suggestion

- [API documentation] (https://help.sap.com/doc/38ea0a832f064060a834a2d328038ceb/1.0/en-US/index.html)

```

File: docs/js/release-notes.mdx (hunk 2 above)
Comment (marikaner) [reply docs/js/release-notes.mdx:53]: Good point.

File: docs/js/release-notes.mdx (hunk 1 above)
Comment (marikaner) [reply docs/js/release-notes.mdx:57]: I thought of this and then forgot. Whoopsie.

File: docs/js/release-notes.mdx [hunk 5]
```diff
@@ -45,6 +45,24 @@ These libraries are not Open Source, but free and distributed under the SAP Deve
...
+- [API documentation] (https://help.sap.com/doc/38ea0a832f064060a834a2d328038ceb/1.0/en-US/index.html)
+
+### Compatibility Notes
+
+- A few operations were removed from the some SAP S/4HANA Cloud 2102 services. These operations were never supported by the SAP S/4HANA system (like CRUD support for some entities), so no existing functionality is affected. The following services are affected:
+  - _Nota Fiscal – Create, Update_ - `API_LOGBR_NOTAFISCAL_SRV`
+  - _Purchase Contracts_ - `API_PURCHASECONTRACT_PROCESS_SRV`
+  - _Outbound Delivery (A2X)_ - `API_OUTBOUND_DELIVERY_SRV_0002`
```
Comment (marikaner) [inline docs/js/release-notes.mdx:57]:
```suggestion
  - _Outbound Delivery (A2X)_ - `API_OUTBOUND_DELIVERY_SRV_0002`
- The service `@sap/cloud-sdk-vdm-purchasing-inforecord-service` was renamed to `@sap/cloud-sdk-vdm-purchasing-info-record-service`.
```

**Review by jjtang1985 — APPROVED** [review]
LGTM

File: docs/js/release-notes.mdx [hunk 6]
```diff
@@ -45,6 +45,25 @@ These libraries are not Open Source, but free and distributed under the SAP Deve
...
```
Comment (artemkovalyov) [inline docs/js/release-notes.mdx:57]:
```suggestion
<!-- vale off -->
  - _Nota Fiscal – Create, Update_ - `API_LOGBR_NOTAFISCAL_SRV`
  - _Purchase Contracts_ - `API_PURCHASECONTRACT_PROCESS_SRV`
  - _Outbound Delivery (A2X)_ - `API_OUTBOUND_DELIVERY_SRV_0002`
  <!-- vale on -->
```

**Comment by artemkovalyov** [comment]
I fixed the build and merging this! Thanks for taking care of the release noted, @marikaner !

**Pull request merged**